members/WIL000450.yaml

```diff
@@ -17,6 +17,9 @@ contact_form:
           selector: div.citystatezipcode input
           value: $ADDRESS_CITY_STATE_ZIP
           required: true
+        - name: phone
+          selector: div.phonecell input
+          value: $PHONE
```

Comment:
required field?

Reply:
good catch. Editing that now

Reply:
looks like it is not required on the form

Reply:
wouldn't we want clients to be able to include phone numbers tho? Or will that be an issue for them cuz they will have to collect phone numbers of supporters? I'm fine with it either way

Reply:
if phone number is set to required: true in a yaml and clients configure the advocacy form so that phone number is disabled, then supporters matched to that target will be prompted to enter a phone number in a special step.

we don't want to trigger that prompt unless phone is actually required to submit the form, since if you ask people to fill in additional information some % will not submit at all, and we want to avoid that.

the yaml should match what is on the form, so if a field is present and not required to submit, it should be there in the yaml with `required: false`

Reply:
perfect! that makes lots of sense. I'll keep a note of that. Thanks!

Reply:
please change this yaml to have the phone field set to `required: false`

Reply:
done!